src/utils/getIsReactionEnabled.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * This function helps consider the every condition
+ * related to enabling emoji reaction feature.
+ */
+
+export interface IsReactionEnabledProps {
+  appLevel?: boolean;
+  isBroadcast?: boolean;
+  isSuper?: boolean;
+  globalLevel?: boolean;
+  moduleLevel?: boolean | null;
+}
+
+export function getIsReactionEnabled({
+  appLevel = true,
+  isBroadcast = false,
+  isSuper = false,
+  globalLevel = true,
+  moduleLevel = null,
+}: IsReactionEnabledProps): boolean {
+  if (moduleLevel) {
+    return !isBroadcast && !isSuper;
+  }
+  return (
+    appLevel
+    && globalLevel
+    && (moduleLevel ?? true)
```

Comment:
Q. what's the reason behind we set the fallback value of moduleLevel as true here?

Reply:
When moduleLevel is undefined so it's null, it shouldn't be falsy